pkgs/development/python-modules/sagemaker/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchPypi, attrs, boto3, google-pasta, importlib-metadata, numpy, protobuf, protobuf3-to-dict, smdebug-rulesconfig }:
+
+buildPythonPackage rec {
+  pname = "sagemaker";
+  version = "2.24.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1j1a058ic00yxnf0cc364fzn82pacih5ffrh5s4dw1q4s3by4cvd";
+  };
+
+  doCheck = false;
+
+  propagatedBuildInputs = [ attrs boto3 google-pasta importlib-metadata numpy protobuf protobuf3-to-dict smdebug-rulesconfig ];
```

getting a little long
```suggestion
  propagatedBuildInputs = [
    attrs
    boto3
    google-pasta
    importlib-metadata
    numpy
    protobuf
    protobuf3-to-dict
    smdebug-rulesconfig
  ];
```